### ISSUE TITLE
[Functional-Social Embedded-Create Profile]:There is no information provided for the input text limit.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#6]
Priority: 

Repro Steps
1. Install "One Bus Away" app on Android.  
2. Start "One Bus Away" app from Android phone.
3. "One Bus Away " Home page will get appear.
4. Navigate to "Open Navigation Drawer" button and press ENTER (or double tap) to open.
5. Navigate to "Sign In" and press ENTER (or double tap) to open. 
6. Navigate to "Sign In with google" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Navigate to “First Name” and enter valid input continuously.
10. Verify that there is any information is provided regarding input text limit.

Expected
Proper information should be provided regarding the input text limit. Screen reader should also convey proper information regarding input text limit.

Actual
There is no information provided for the input text limit. Screen reader does not convey any information regarding input text limit, while there is input text have an limit.

MAS Rule: Functional

Impact
User who is rely on screen reader will get confuse and face difficulty if there is no information is provided for input text limit.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
